examples/resource-class/nvidia.tesla.gpu.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Device
+metadata:
+  name: nvidia-tesla-gpu
+  labels:
+    type: nvidia-gpu
+    quantity: "8"
+    compute-ability: "3.7"
```

Comment:
[root@dell-r620-01 kubernetes]# kubectl get resourceclass my.rc2 -o yaml
apiVersion: v1
kind: ResourceClass
metadata:
  creationTimestamp: 2017-05-30T05:27:48Z
  name: my.rc2
  resourceVersion: "15222"
  selfLink: /api/v1/namespaces/default/resourceclasses/my.rc2
  uid: b820fe43-44f8-11e7-94f7-bc305bf4e400
spec:
  resourceSelector:
  - matchExpressions:
    - key: Type
      operator: In
      values:
      - nvidia-gpu
status: {}